### ISSUE TITLE
Balanced backstab formula (under 10 players)

### ISSUE THREAD

**Issue by Spyper12** [50DKP/FF2-Official#292]
If there is 10> players backstab formula should be 1/n
9 players = backstab damage : 1/9  hp of boss
8 players = backstab damage : 1/8 hp of boss
2 players = backstab damage : 1/2 hp of boss
1 player   = backstab damage :  1/1 = all hp of boss

**Comment by Starblaster64** [comment]
I'm going to assume that the number of players in each of your examples are the number of players on the non-Boss team and not the total number of players active on the server. (Hence the following notation `(n-1)⁻¹` where n = total number of active players)

(10-1)⁻¹ would actually result in a lower backstab damage (11.1%) than the current 12%, and (9-1)⁻¹ would only increase it to 12.5%. (8-1)⁻¹ and beyond see much more noticeable effects on damage output (~14.3%), so if such a system were to be implemented it would be best to only have it take effect from 8 players and below (using the current backstab formula).

Personally I think the VSH backstab formula should be adapted in some way, albeit tweaked somewhat to accommodate very high and/or very low health bosses.

**Comment by Hutgib1** [comment]
If it takes effect when there are 8 players or below, it is okay. 9 or 10 players haven't got much difference. I think this is a good idea. Being able to use spy when there are small amount of players. Spy is becoming useless under 10-8 players. So, I support this.

**Comment by Jelly120** [comment]
I agree. I support this ldea too. Let spy be balanced under 10 players.
